portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsBooleanField.testcase

```diff
@@ -0,0 +1,413 @@
+
+		FormFields.viewBooleanField(fieldValue = "Copy of Boolean");
+
+		FormViewBuilder.assertHelpTextPresent(
+			fieldName = "Boolean",
+			fieldValue = "Test Help Text");
+
+		FormViewBuilder.assertHelpTextPresent(
+			fieldName = "Copy of Boolean",
+			fieldValue = "Test Help Text");
+	}
+
+	@description = "Verify that a form with the boolean field can be required"
+	@priority = "5"
+	test CanBeRequired {
+		FormsAdmin.addForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Boolean");
+
+		FormFields.viewBooleanField(fieldValue = "Boolean");
+
+		FormFields.enableSwitch(fieldName = "required");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormFields.viewRequiredAsterisk();
+
+		FormPortlet.submitWithValidationError(validationErrorMessage = "This field is required.");
+
+		Check.checkToggleSwitch(
+			fieldName = "Boolean",
+			locator1 = "FormFields#CHECK_BOOLEAN_FIELD");
+
+		FormPortlet.submitSuccessfully();
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoViewEntries();
+
+		FormsAdmin.viewFormTableEntry(
+			fieldLabel = "Boolean",
+			fieldValue = "True");
+	}
+
+	@description = "Verify that a form with the boolean field can be set to show as a switch"
+	@priority = "5"
+	test CanBeSetToShowAsASwitch {
+		FormsAdmin.addForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Boolean");
+
+		FormFields.viewBooleanField(fieldValue = "Boolean");
+
+		FormFields.enableSwitch(fieldName = "Show as a Switch");
+
+		FormFields.viewSwitch(fieldName = "Boolean");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormFields.viewSwitch(fieldName = "Boolean");
+
+		FormFields.enableSwitch(fieldName = "Boolean");
+
+		FormPortlet.submitSuccessfully();
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoViewEntries();
+
+		FormsAdmin.viewFormTableEntry(
+			fieldLabel = "Boolean",
+			fieldValue = "True");
+	}
+
+	@description = "Verify that a form with the boolean field can be submitted"
+	@priority = "5"
+	test CanBeSubmitted {
+		FormsAdmin.addForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Boolean");
+
+		FormFields.editText(
+			fieldName = "label",
+			fieldValue = "Boolean 1");
+
+		Form.closeSidebar();
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Boolean");
+
+		FormFields.editText(
+			fieldName = "label",
+			fieldValue = "Boolean 2");
+
+		FormFields.viewBooleanField(fieldValue = "Boolean 1");
+
+		FormFields.viewBooleanField(fieldValue = "Boolean 2");
+
+		FormViewBuilder.gotoFirstFieldset(fieldLabel = "Boolean 1");
```

Comment:
@rafaellajordao, shouldn't we use a different macro to select the first Boolean Field?

The macro `gotoFirstFieldset` makes me understand that the test is actually manipulating a Fields Group (a cousin of Field Set) instead of two separated fields. Please take a look below on the images to see the difference between fields in and out Fields Group: 

- Boolean Field in a Fields Group
![image](https://user-images.githubusercontent.com/6729287/136096121-8ad8249a-07ff-48b1-9d65-ef8912daf398.png)

- Boolean Field out a Fields Group
![image](https://user-images.githubusercontent.com/6729287/136096024-472db98d-2458-48ff-8dba-03eb8ebf7946.png)